android/keyframes/src/main/java/com/facebook/keyframes/KeyframesDrawableAnimationCallback.java

```diff
@@ -105,6 +105,17 @@ public void start() {
   }
 
   /**
+   * Starts the animation and plays it once
+   */
+  public void playOnce() {
+    mStopAtLoopEnd = true;
+    mStartTimeMillis = 0;
+    mCurrentLoopNumber = 0;
```

Comment:
should this be init'd to -1?

looks like below there's a `loopHasEnded = currentLoopNumber > mCurrentLoopNumber;`